lib/core_to_mucore.ml

```diff
@@ -1378,8 +1452,10 @@ module Spec = struct
   type desugared =
     { trusted : Mu.trusted;
       accesses : (Cerb_location.t * (Sym.t * Ctype.ctype)) list;
-      requires : (Sym.t, Ctype.ctype) Cn.cn_condition list;
-      ensures : (Sym.t, Ctype.ctype) Cn.cn_condition list;
+      requires :
+        (Sym.t * Sym.t Cn.cn_base_type) list * (Sym.t, Ctype.ctype) Cn.cn_condition list;
+      ensures :
+        (Sym.t * Sym.t Cn.cn_base_type) list * (Sym.t, Ctype.ctype) Cn.cn_condition list;
```

Comment:
Please use extra named fields instead of tuples, e.g. `ghost_param` or `ghost_args` and `ghost_return`.